providers/common/include/internal/core_mkdigest.h

```diff
@@ -42,6 +42,19 @@ static void *name##_dupctx(void *ctx) \
 
 # define OSSL_FUNC_DIGEST_GET_PARAM(name, blksize, dgstsize, flags)     \
 static OSSL_OP_digest_get_params_fn name##_get_params;                  \
+static OSSL_OP_digest_gettable_types_fn name##_gettable_types;          \
+static const OSSL_PARAM known_##name##_gettable_types[] = {             \
+    {OSSL_DIGEST_PARAM_BLOCK_SIZE, OSSL_PARAM_INTEGER,                  \
```

Comment:
Why are you macroing a const block?. This can be shared by all without duplicating it.

Reply:
This whole file could do with some refactoring, but that's for another PR